dali/test/python/test_operator_crop_mirror_normalize.py

```diff
@@ -64,41 +64,6 @@ def define_graph(self):
         images = self.cmn(images, mirror=rng)
         return images
 
-def check_cmn_cpu_vs_gpu(batch_size, dtype, output_layout, mirror_probability, mean, std, scale, shift, pad_output):
```

Comment:
this test is redundant. Same thing is tested vs a python implementation. There's no need to test cpu vs gpu here.